src/dmd/tokens.d

```diff
@@ -94,18 +94,6 @@ enum TOK : int
     index,
     is_,
 
-    // 64
-    // NCEG floating point compares
-    // !<>=     <>    <>=    !>     !>=   !<     !<=   !<>
-    unord,
-    lg,
-    leg,
-    ule,
-    ul,
-    uge,
-    ug,
-    ue,
-
     // 72
```

Comment:
I guess this number should be removed too as it gets outdated too easily.

Reply:
I subtracted 8 from each of the numbers to try and keep the status quo.